config/crd/bases/cellular.config.nephio.org_configs.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.13.0
+  name: configs.cellular.config.nephio.org
+spec:
+  group: cellular.config.nephio.org
```

Comment:
Change group to --> `nf.nephio.org` and kind to `Config` according to the discussion with @henderiw and @josephthaliath because it allows extending to other type of network functions except cellular network functions.